### PR TITLE
Bazel build fixes.

### PR DESCRIPTION
**Pull request by asa**
These are a few fixes I needed to get the included bazel file to play nice with OpenImageIO.
I believe they are general enough to be included for others.
I squashed the previous PR (#1816 )work into this new branch and pushed it here instead.

### CODE REVIEW

**Comment by asa** [comment]
@Vertexwahn @cary-ilm  can I get a review?
Had to make the new pr as the rebase pulled in new commits over the top of the ones I needed to sign. This is a clean, single commit of the same thing.

File: cmake/OpenEXRConfig.h.in [hunk 1]
```diff
@@ -17,9 +17,9 @@
 //
 
 #ifndef INCLUDED_OPENEXR_VERSION_H
-#define OPENEXR_VERSION_MAJOR @OpenEXR_VERSION_MAJOR@
```
Comment (cary-ilm) [inline cmake/OpenEXRConfig.h.in:20]: I'm curious what prompted this change? It should be benign, as I believe OPENEXR_VERSION_* and OpenEXR_VERSION_* have the same values. Not opposed to it, just wondering why it should change.

**Review by cary-ilm — APPROVED** [review]
LGTM, with one question about the change to OpenEXRConfig.h.in

File: cmake/OpenEXRConfig.h.in (hunk 1 above)
Comment (asa) [reply cmake/OpenEXRConfig.h.in:20]: When migrating to bazel we take and make bazel versions of these macros, and having them all consistently upper case seams cleaner for these kinds of external automation.

**Pull request merged**